client/my-sites/plans-features-main/wpcom-faq.jsx

```diff
@@ -73,7 +73,7 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<FAQItem
 				question={ translate( 'Can I install my own theme?' ) }
 				answer={ translate(
-					"Yes! With the WordPress.com Business plan you can install any theme you'd like." +
+					"Yes! With the WordPress.com Business or eCommerce plan you can install any theme you'd like." +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Yes! With the WordPress.com Business plan you can install any theme you'd like. All plans give you access to our {{a}}directory of free and premium themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/34722)